core/base/inc/LinkDef.h

```diff
@@ -1,16 +1,16 @@
-#include "core/base/inc/LinkDef1.h"
-#include "core/base/inc/LinkDef2.h"
-#include "core/base/inc/LinkDef3.h"
-#include "core/cont/inc/LinkDef.h"
-#include "core/meta/inc/LinkDef.h"
+#include "./LinkDef1.h"
+#include "./LinkDef2.h"
+#include "./LinkDef3.h"
+#include "../../cont/inc/LinkDef.h"
+#include "../../meta/inc/LinkDef.h"
 
 #if defined(SYSTEM_TYPE_winnt)
-#include "core/winnt/inc/LinkDef.h"
+#include "../../winnt/inc/LinkDef.h"
```

Comment:
Interesting if there is a way not to use relative paths here?

Reply:
One can add extra include path with `${CMAKE_SOURCE_DIR}`

Reply:
these will be amazing!

Reply:
Wouldn't it be easier to just merge all `LinkDef.h` for core/ into `core/inc/LinkDef.h`? What stops us from doing that?

Reply:
Possibly, and I tried some of that in the past. And it can happen independently from this PR: this PR does not make it worse (nor does it improve it).